### PR TITLE
Remove color in CI

### PR DESCRIPTION
**Pull request by clee2000**
Remove color by default to make CI logs easier to read

Example of color
image

### CODE REVIEW

File: test/test_utils.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ def _check_checkpoint_sequential(
     # Test whether checkpoint is being triggered or not. For this, we check
     # the number of times forward pass happens
     def test_checkpoint_trigger(self):
+        self.assertEqual(1, 2)
```
Comment (ZainRizvi) [inline test/test_utils.py:119]: accidentally left in?

**Review by ZainRizvi — COMMENTED** [review]
Does this mean that colors will only show up when these commands are run locally but never in github actions logs?

**Review by malfet — COMMENTED** [review]
I think it should be controlled by terminal setting, rather than whether or not something is running in CI (may be by calling something like `stdout.isatty()`)

**Comment by kit1980** [comment]
> I think it should be controlled by terminal setting, rather than whether or not something is running in CI (may be by calling something like `stdout.isatty()`)

Yeah, like you added to torchfix https://github.com/pytorch-labs/torchfix/blob/main/torchfix/common.py#L11

**Comment by ZainRizvi** [comment]
> I think it should be controlled by terminal setting, rather than whether or not something is running in CI (may be by calling something like `stdout.isatty()`)

Does this mean that it would auto-detect if it's in a terminal that can output color properly?

**Comment by clee2000** [comment]
I'm just going to remove color in CI, determining if it should be colored according to terminal can come later

**Review by ZainRizvi — APPROVED** [review]

**Pull request closed without merging**